src/docs/self-hosted/productionalizing.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: 'Self-Hosted Productionalizing'
+---
+
+This page provides a guide to help users secure their Sentry deployments and make the most out of Sentry. For further guide relating system security and performance, you might want to 
+
+## Recommended system resource
+
+Depending on your usage, required system resource to run Sentry may varies. Minimum system specification for running Sentry looks like this:
+
+- 2 CPU cores
+- 4 GB RAM
+
+Having a machine that have lower system specification than that will fail the installation script.
+
+Current recommended system specification for running Sentry is:
+
+- 4 CPU cores
+- 16 GB RAM
+- 20 GB free disk storage space
+
+Depending on your load, you might want to increase your system specification to handle higher traffic load better. If increasing the disk storage space isn't possible, you can migrate your storage to use external storage such as AWS S3 or Google Cloud Storage (GCS). Decreasing your `SENTRY_RETENTION_DAYS` environment variable to lower numbers will save some storage space from being full, at the cost of having shorter data retention period.
+
+## Enabling HTTPS
+
+We recommend TLS termination to be done on your own dedicated load balancer. Although you can set it on the `nginx.conf` file, it is not recommended as newer self-hosted release might alter some configuration on the file. Some example are available on [Using Dedicated Load Balancer](#using-dedicated-load-balancer) section.
```

Comment:
```suggestion
We recommend TLS termination to be done on your own dedicated load balancer. Although you can set it on the `nginx.conf` file, it is not recommended as newer self-hosted releases might alter some configurations on the file. Some examples are available on [Using Dedicated Load Balancer](#using-dedicated-load-balancer) section.
```

Reply:
> Although you can set it on the `nginx.conf` file, it is not recommended as newer self-hosted release might alter some configuration on the file.

Users can also configure settings on the host nginx config, not the config that comes with the self-hosted setup. That's actually what we do, so not sure if this is completely accurate.